renderapi/image.py

```diff
@@ -51,6 +52,8 @@ def get_bb_image(stack, z, x, y, width, height, scale=1.0,
         number of units @scale=1.0 down (+y) of bounding box to render
     scale : float
         scale to render image at (default 1.0)
+    channel : str
+        channel name to render

```

Comment:
Render's handling of channel names and weight is pretty unintuitive -- could we document the fact that it can be "channel1__weight1__channel2__weight2..." and also change the name to channels to reflect the REST API?

Reply:
agreed... i improved the documentation.  You can just list single channel names, and really I think that's the only way I will use it.  But I've added the documentation about how to format it if you want the weighted channels.